addons/website_project_issue/controllers/main.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import werkzeug
+
+from openerp import http
+from openerp.http import request
+
+from openerp.addons.website_portal.controllers.main import website_account
+
+class WebsiteAccount(website_account):
+    @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
+    def account(self):
+        response = super(WebsiteAccount, self).account()
+        project_issues = request.env['project.issue'].search([])
```

Comment:
Visibility of issues are managed using defined [record rules](https://github.com/odoo-dev/odoo/blob/master-website_project_issue-portalview-dja/addons/website_project_issue/security/portal_security.xml#L5).